webcomponents/input/input.webcomponent.spec.js

```diff
@@ -176,11 +176,30 @@ describe('mn-input (webcomponent)', () => {
     })
 
     it('should set empty text to label when attribute is removed', () => {
-      element.setAttribute('placeholder','test')
+      element.setAttribute('placeholder', 'test')
       element.removeAttribute('placeholder')
       expect(element).to.contain('label').with.text('')
     })
   })
+
+  describe('attribute maxlength', () => {
+    it('should define attribute in child input', () => {
+      element.setAttribute('maxlength', '2')
+      expect(element).to.contain('input').with.attribute('maxlength', '2')
+    })
+
+    it('should update attribute in child input', () => {
+      element.setAttribute('maxlength', '2')
+      element.setAttribute('maxlength', '3')
+      expect(element).to.contain('input').with.attribute('maxlength', '3')
+    })
+
+    it('should remove attribute from child input', () => {
+      element.setAttribute('maxlength', '2')
+      element.removeAttribute('maxlength')
+      expect(element).to.contain('input').to.not.have.attribute('maxlength')
```

Comment:
This verification is wrong, because maxlength attribute exists and the test is passing.